sys-apps/ignition/ignition-9999.ebuild

```diff
@@ -11,7 +11,7 @@ inherit coreos-go cros-workon systemd udev
 if [[ "${PV}" == 9999 ]]; then
 	KEYWORDS="~amd64 ~arm64"
 else
-	CROS_WORKON_COMMIT="6a51b2f37ffec93c3a378578e14163f370585207" # tag v0.18.0
+	CROS_WORKON_COMMIT="b036c2d1a769bf4eca8b72cc3cd750d8f2fa5058" # tag v0.19.0
```

Comment:
What is this SHA1 from?  The tag is `01c039a5ce59acd39e5741713e59abfcb74d0782`.

Reply:
Was on a different branch, didn't realize. Updated to be correct.

Reply:
It still has the bad value, unless GitHub is caching across page refreshes.

Reply:
Sorry about that. Forgot to `git add` it.